### ISSUE TITLE
Onefuzz deployment throws system exception at the end

### ISSUE THREAD

**Issue by nageshn66** [microsoft/onefuzz#1529]
## Information

- Onefuzz version: 4.0.0
- OS:Ubuntu Linux

## Provide detailed reproduction steps (if any)

1. Calling deploy.py script with necessary args including --client_id and --client_secret
2. just before deployment completes, I see this exception.
    INFO:deploy:deploying function app api-service.zip
    System.Exception: Cannot find the Local Application Data.
   at Azure.Functions.Cli.Utilities.EnsureCoreToolsLocalData() in D:\a\1\s\src\Azure.Functions.Cli\Common\Utilities.cs:line 180
   at Azure.Functions.Cli.Telemetry.PersistenceChannel.StorageService.VerifyOrSetDefaultStorageDirectoryPath(String desireStorageDirectoryPath) in D:\a\1\s\src\Azure.Functions.Cli\Telemetry\PersistenceChannel\StorageService.cs:line 96
   at Azure.Functions.Cli.Telemetry.PersistenceChannel.StorageService.Init(String storageDirectoryPath) in D:\a\1\s\src\Azure.Functions.Cli\Telemetry\PersistenceChannel\StorageService.cs:line 78
   at Azure.Functions.Cli.Telemetry.PersistenceChannel.PersistenceChannel..ctor(String storageDirectoryPath, Int32 sendersCount) in D:\a\1\s\src\Azure.Functions.Cli\Telemetry\PersistenceChannel\PersistenceChannel.cs:line 42
   at Azure.Functions.Cli.Telemetry.Telemetry.InitializeTelemetry() in D:\a\1\s\src\Azure.Functions.Cli\Telemetry\Telemetry.cs:line 93

### Expected result
Deploy without system exception 

_What is the expected result of the above steps?_

### Actual result

_What is the actual result of the above steps?_

**Comment by ranweiler** [comment]
Duplicate of #899.

**Comment by nageshn66** [comment]
Hi Joe,  I see this closed as duplicate. Is there an issue similar opened already? is there a fix for this? thanks

**Comment by nageshn66** [comment]
Thanks @ranweiler , I see the notes behind the issue, that helps to fix the issue.

**Comment by ranweiler** [comment]
(@nageshn66, I actually hadn't closed this yet, just triaged it as a duplicate)

The issue is upstream in the `azure-functions-core-tools-3`, which is used to deploy the Function App.

If you're unblocked, please feel free to close this issue out!

**Comment by nageshn66** [comment]
Thanks @ranweiler, got it. Closing this issue for now as it unblocked now by setting the default to XDG_DATA_HOME variable.
